Formula/swi-prolog.rb

```diff
@@ -69,7 +69,7 @@ def install
     # './prepare' prompts the user to build documentation
     # (which requires other modules). '3' is the option
     # to ignore documentation.
-    system "echo '3' | ./prepare" if build.head?
+    system "echo", "'3' | ./prepare" if build.head?
```

Comment:
Nice, Thank you for fixing that ❤️. I'd put it on my list to look at this afternoon and it's already 5pm and I hadn't yet done so, sigh.